components/Meta.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import { SITE_DESCRIPTION, SITE_NAME } from "@/constants.ts";
+
+interface MetaProps {
+  title?: string;
+  description?: string;
+  imageUrl?: string;
+  url?: string;
```

Comment:
Are you able to change this to `href`? It might be confusing since the value passed to this parameter is `URL.href`.